### PR TITLE
Get example from the schema when available

### PR DESCRIPTION
**Pull request by olivier-heurtier**

### CODE REVIEW

File: sphinxcontrib/openapi/openapi30.py [hunk 1]
```diff
@@ -119,6 +118,10 @@ def _parse_schema(schema, method):
         return [_parse_schema(schema['items'], method)]
 
     if schema_type == 'object':
+        if 'example' in schema:
+            example = schema.get('example')
+            return collections.OrderedDict(example)
```
Comment (ikalnytskyi) [inline sphinxcontrib/openapi/openapi30.py]: According to OpenAPI 3.0 spec, the `example` of schema:

> A free-form property to include an example of an instance for this schema. To represent examples that cannot be naturally represented in JSON or YAML, a string value can be used to contain the example with escaping where necessary.

Which means, we cannot cast it to OrderedDict.

File: tests/test_openapi.py [hunk 2]
```diff
@@ -1146,6 +1146,109 @@ def test_callback(self):
...
+                  Something
+
+                  **Example response:**
+
+                  .. sourcecode:: http
+
+                     HTTP/1.1 200 OK
+                     Content-Type: application/json
+
+                     "A sample"
```
Comment (ikalnytskyi) [inline tests/test_openapi.py]: It should not contain quotes, as far as I understand. Schema's `example` may contain JSON/YAML as text, and that text should be propagated to the snippet with no quotes.

**Comment by ikalnytskyi** [comment]
@olivier-heurtier thanks for your work! I'm sorry for the delay in my review. I like the change, yet, I think we need to fix the issue (see my comments inline) before we can land it.

**Comment by olivier-heurtier** [comment]
Changes made as requested.

File: sphinxcontrib/openapi/openapi30.py [hunk 3]
```diff
@@ -114,6 +114,10 @@ def _parse_schema(schema, method):
         return [_parse_schema(schema['items'], method)]
 
     if schema_type == 'object':
+        if 'example' in schema:
+            example = schema.get('example')
```
Comment (ikalnytskyi) [inline sphinxcontrib/openapi/openapi30.py]: As far as I understand, `example` property is not tied to `schame_type == 'object'`, i.e. the property may be defined for any type.

**Comment by ikalnytskyi** [comment]
Nice! Thank you!

**Pull request merged**